src/app/code/community/Omise/Gateway/Model/Api/Charge.php

```diff
@@ -161,9 +161,7 @@ public function isUnauthorized()
      */
     public function isPaid()
     {
-        $paid = isset($this->paid) ? $this->paid : $this->captured;
```

Comment:
why that is no more necessary?

Reply:
To be honest, I can't remember - it is in a commit titled 'Fixes CC payments to work with 2017 API' - so I guess it relates to that